pom.xml

```diff
@@ -40,6 +40,34 @@
     </dependencyManagement>
 
     <dependencies>
+        <!--https://stackoverflow.com/questions/44088493/jersey-stopped-working-with-injectionmanagerfactory-not-found-->
```

Comment:
Dette er noe jeg vil snakke med deg om @runeflobakk. Dette måtte til for at jeg skulle få biblioteket til å fungere i tester. Hvor skal det legges? Ser noe av det bør inn i api-specification, men dette er jeg veldig usikker på hvorfor er sånn.

Reply:
Den trengtes for å kjøre tester, altså runtime. Men nå når jeg fjernet den så gjelder det ikke mer. Så 🔪 den vekk.